doc/api/errors.md

```diff
@@ -253,6 +256,44 @@ will affect any stack trace captured _after_ the value has been changed.
 If set to a non-number value, or set to a negative number, stack traces will
 not capture any frames.
 
+### `error.cause`
+
+<!-- YAML
+added: v16.9.0
+-->
+
+* {any}
+
+The `error.cause` property is the cause of the error, as set by calling
+`new Error(message, { cause })`. It is `undefined` if no cause has been set.
```

Comment:
There's a nuance here: if no cause has been set, there won't be any `cause` property at all. `error.cause` will be `undefined` if the cause of the error is itself `undefined`.

```js
console.log('cause' in new Error); // false
console.log('cause' in new Error('', {cause:undefined})); // true
```

```suggestion
The `error.cause` property is the cause of the error, as set by calling
`new Error(message, { cause })`. It is not set if no cause has been set.
```

Reply:
Sure that would work!